src/Models/User.php

```diff
@@ -72,4 +72,14 @@ public function getGravatarAttribute($size = 200)
     {
         return sprintf('https://www.gravatar.com/avatar/%s?size=%d', md5($this->email), $size);
     }
+
+    /**
+     * Returns whether a user is at admin level.
+     *
+     * @return bool
+     */
+    public function getIsAdminAttribute()
+    {
+        return $this->level === 1;
```

Comment:
Yeh, but I've had issues with things like this before where I thought they were ints, but they were actually strings. Maybe try var_dumping it to see what it actually is. :)